src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,16 @@
-package core.basesyntax;
-
-public class Application {
-    public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
-    }
-}
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        // create three balls using class Lottery and print information about them in console
+

```

Comment:
```suggestion
```

Reply:
I try to do it in such way:

        Lottery lotteryBall = new Lottery();
        Ball [] setOfBalls =
                {lotteryBall.getRandomBall(), 
                 lotteryBall.getRandomBall(),
                 lotteryBall.getRandomBall()};

but I always got the same ball.
then i have to update now class Lottery: creation of Ball-object i should relocate in method getRandomBall()
I will make it now